diff --git a/core/CMakeLists.txt b/core/CMakeLists.txt
index 11096a0..10d9000 100644
--- a/core/CMakeLists.txt
+++ b/core/CMakeLists.txt
@@ -20,7 +20,7 @@ add_library(${PROJECT_NAME}::core ALIAS core)
 target_link_libraries(core
   PRIVATE
     Threads::Threads
-    $<$<AND:$<BOOL:UNIX>,$<NOT:$<BOOL:APPLE>>>:rt>
+    rt
 )
 
 if(HAVE_CXX_LIBATOMIC)
diff --git a/pull/CMakeLists.txt b/pull/CMakeLists.txt
index 6dfa6a2..fca7b42 100644
--- a/pull/CMakeLists.txt
+++ b/pull/CMakeLists.txt
@@ -41,7 +41,7 @@ target_link_libraries(pull
   PRIVATE
     Threads::Threads
     $<IF:$<BOOL:${USE_THIRDPARTY_LIBRARIES}>,${PROJECT_NAME}::civetweb,civetweb::civetweb-cpp>
-    $<$<AND:$<BOOL:UNIX>,$<NOT:$<BOOL:APPLE>>>:rt>
+    rt
     $<$<BOOL:${ENABLE_COMPRESSION}>:ZLIB::ZLIB>
 )
 
diff --git a/push/CMakeLists.txt b/push/CMakeLists.txt
index 81e606f..e9f5492 100644
--- a/push/CMakeLists.txt
+++ b/push/CMakeLists.txt
@@ -13,7 +13,7 @@ target_link_libraries(push
   PRIVATE
     Threads::Threads
     CURL::libcurl
-    $<$<AND:$<BOOL:UNIX>,$<NOT:$<BOOL:APPLE>>>:rt>
+    rt
 )
 
 target_include_directories(push
